docs/release-notes/8.16.asciidoc

```diff
@@ -9,6 +9,63 @@
 [[known-issue-8.16.1]]
 ==== Known issues
 
+// tag::known-issue[201820]
+[discrete]
+.The **Exceptions** tab won't properly load if exceptions contain comments with newline characters (`\n`)  
+[%collapsible]
+====
+*Details* +
+On December 5, 2024, it was discovered that the **Exceptions** tab will not load properly if any exceptions contain comments with newline characters (`\n`). This issue occurs when you upgrade to 8.16.0 or later. 
+
+*Workaround* + 
+
+For custom rules:
+
+. From the {security-app}, <<import-export-rules-ui,export>> the rule or rules with the affected exception lists. 
+. Modify the `.ndjson` file so `comments` no longer contain newline characters.
+. Return to the {security-app} and <<import-export-rules-ui,re-import>> the rules. Make sure to select the **Overwrite existing exception lists with conflicting "list_id"** option.
+
+For prebuilt rules: 
+
+. Fetch the affected exception list ID or IDs that are associated with the rule. 
+.. Find the affected rule's ID (`id`). From the {security-app}, open the rule's details page, go to the page URL, and copy the string at the end. For example, the URL http://host.name/app/security/rules/id/167a5f6f-2148-4792-8226-b5e7a58ef46e contains the `id` `167a5f6f-2148-4792-8226-b5e7a58ef46e` at the end.
+.. Use the `id` to fetch the rule's details using the {api-kibana}/operation/operation-readrule[Retrieve a detection rule API]. Here is an example request that includes the rule ID:
```

Comment:
@natasha-moore-elastic I could use your help here 😅 I'm fairly certain that the attribute is correct, but I don't know how to form the second half of the reference. Here are the ESS API docs that I need to reference: https://www.elastic.co/docs/api/doc/kibana/v8/operation/operation-readrule

```suggestion
.. Use the `id` to fetch the rule's details using the {api-kibana}/operation/operation-readrule[Retrieve a detection rule API]. Here is an example request that includes the rule ID:
```

I'll need to do this again for the Serverless changelog [here](https://github.com/elastic/docs-content/pull/233/files#diff-e0d59f20eb51b705c21eb9969115968e0f96c1815e4107c1f237cf2cbbe7aa4cR36). For those release notes, I'll need to reference [these](https://www.elastic.co/docs/api/doc/serverless/operation/operation-readrule) Serverless API docs.

Reply:
Hey @nastasha-solomon, your reference to the ESS Retrieve a detection rule API looks good to me and works as expected in the preview! 🚀 

For serverless Kibana API docs, I can't see an existing attribute anywhere in the `docs` repo, so I opened a [PR](https://github.com/elastic/docs/pull/3137) to add one. When this is merged, you can use similar syntax to refer to the serverless Retrieve a detection rule API:

`{api-kibana-serverless}/operation/operation-readrule[Retrieve a detection rule API]`